From b692fd86471c8b3c5e0e92007b7b2d5cfd058af7 Mon Sep 17 00:00:00 2001
From: Kouhei Sutou <kou@clear-code.com>
Date: Sat, 13 Apr 2013 19:19:36 +0900
Subject: [PATCH] actor: fix clutter-win32.h include path

It is installed under "clutter/win32/" directory. Other include paths
were fixed in 9da7dd191dae7d1cc626ea22998145ebc2bb0722 but it was not
fixed.
---
 clutter-gtk/gtk-clutter-actor.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clutter-gtk/gtk-clutter-actor.c b/clutter-gtk/gtk-clutter-actor.c
index f95caed..67d9cdd 100644
--- a/clutter-gtk/gtk-clutter-actor.c
+++ b/clutter-gtk/gtk-clutter-actor.c
@@ -59,7 +59,7 @@
 #endif
 
 #ifdef CLUTTER_WINDOWING_WIN32
-#include <clutter/clutter-win32.h>
+#include <clutter/win32/clutter-win32.h>
 #endif
 #ifdef GDK_WINDOWING_WIN32
 #include <gdk/gdkwin32.h>
-- 
1.7.10.4

